packages/docs/docs/charting/index.md

```diff
@@ -14,6 +14,14 @@ Here is a **sample** of what a charting experience might look like - to be clear
 
 ![Chart sample](/img/tutorials/chart-sample.png)
 
+## Data Model
+
+![Chart Data Model 1](/img/tutorials/charting-annotation-1.png)
+
```

Comment:
As the images don't contain hyperlinks, request to add a sentence below the chart.  You can read more about [Patient](/docs/api/fhir/resources/patient), [Appointment], [AllergyIntolerance]... in the reference material.